packages/docs/content/blog/release-6.0.mdx

```diff
@@ -54,15 +54,15 @@ In our [3.6.0 release](https://github.com/CMSgov/design-system/releases/tag/%40c
   <img src={swatches} alt="Color ramps from the 'System Color Tokens' page, showing a range of light to dark shades of a color" className="ds-u-display--inline-block" width="400" />
 </div>
 
-In this release we've added a page to the design system to show all color tokens in an easy-to-read and scannable way. We're exposing this base set of tokens to better highlight the design decisions made within a particular theme. Think of them as [a complete box of crayons](https://design.cms.gov/foundation/system-color-tokens/?theme=core). Our themes make use of certain colors from the crayon box but not all of them.
+In this release we've added a page to the design system to show all color tokens in an easy-to-read and scannable way. We're exposing this base set of tokens to better highlight the design decisions made within a particular theme. Think of them as [a complete box of crayons](/foundation/system-color-tokens/?theme=core). Our themes make use of certain colors from the crayon box but not all of them.
```

Comment:
is "?theme=core" going to redirect ppl to core even if they're reading this post on Medicare or something?